### DISCUSSION TITLE
Comunicare l'adesione "ufficiale" dei comuni a informa-covid19

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/informa-covid19#40] [Idee/Proposte]
Ciao,
pensavo a due cose:

- predisporre un "comunicato tipo", da mettere a disposizione dei comuni che hanno aderito e aderiranno, per lanciare (se vorranno) nei loro canali di comunicazione la cosa;
- predisporre un badge di adesione [![](https://img.shields.io/badge/informa-COVID19-%23F50057)](https://informacovid.opendatasicilia.it/)

Che ne pensate?

```
 [![](https://img.shields.io/badge/informa-COVID19-%23F50057)](https://informacovid.opendatasicilia.it/)
```

**Comment by gpirrotta** [comment]
Ottima idea, bella l'idea del badge!
